repairnator/repairnator-realtime/src/main/java/fr/inria/spirals/repairnator/realtime/RTScanner.java

```diff
@@ -124,12 +134,12 @@ public void launch() {
             new Thread(this.inspectJobs).start();
             this.running = true;
 
-            if (this.duration != null) {
+            if (this.config.getDuration() != null) {
                 InspectProcessDuration inspectProcessDuration;
                 if (this.endProcessNotifier != null) {
-                    inspectProcessDuration = new InspectProcessDuration(this.duration, this.inspectBuilds, this.inspectJobs, this.buildRunner, this.endProcessNotifier);
+                    inspectProcessDuration = new InspectProcessDuration(this.config.getDuration(), this.inspectBuilds, this.inspectJobs, this.buildRunner, this.endProcessNotifier);
```

Comment:
and we should be able to remove this parameter too

Reply:
Done.